classes/TaxonomyUtilities.php

```diff
@@ -165,6 +165,25 @@ public static function parseScientificName($inStr, $conn = null, $rankId = 0, $k
 						if(!isset($retArr['rankid']) || !$retArr['rankid']) $retArr['rankid'] = 220;
 					}
 				}
+				//Check the retArr[author] array for cultivar epithet, tradename, author
+				if (preg_match('/["]([^"]+)["]/', $retArr['author'], $matches)) {
```

Comment:
I know I'm very late to this, but shouldn't this be looking for single quotes rather than double quotes? Was it tested?

Reply:
I thought the cultivar epithet is in double quotes, isn't it?

Reply:
Nope. Single quotes. E.g. _Acer rubrum_ 'cultivarName' TRADENAME.

Reply:
Great catch then! I can fix that!